src/main/java/net/datafaker/Faker.java

```diff
@@ -647,9 +647,7 @@ public GameOfThrones gameOfThrones() {
         return getProvider(GameOfThrones.class, () -> new GameOfThrones(this));
     }
 
-    public GarmentSize garmentSize() {
-    return getProvider(GarmentSize.class, () -> new GarmentSize(this));
-  }
+    public GarmentSize garmentSize() { return getProvider(GarmentSize.class, () -> new GarmentSize(this)); }
```

Comment:
I think multi-line would be better in order to maintain codestyle

Reply:
okey, but you have many examples with one line

Reply:
@p4pupro No, IDE collapses code to make classes more readable

Reply:
Okey, I did the change to multi line.